dali/operators/image/paste/multipaste.cc

```diff
@@ -19,44 +19,83 @@
+                         nullptr, true, true)
+    .AddOptionalArg<float>("in_anchors_rel", R"code(Relative coordinates of LU corner
+of the source region.
+
+The argument works like `in_anchors`, but the values should be floats in `[0, 1]` range,
+describing the anchor placement relative to the input sample shape.)code",
+                           nullptr, true, true)
+    .AddOptionalArg<int>("shapes", R"code(Shape of the paste regions.
 
 The shapes are represented as 2D tensors where the first dimension corresponds to the
-elements of ``in_ids`` and the second one is equal to the number of dimensions of the
+number of pasted regions and the second one is equal to the number of dimensions of the
 data, excluding channels.
 
-If not provided, the shape is calculated so that the region goes from the region anchor
- in the input image until the end of the input image.)code", nullptr, true)
-.AddOptionalArg<int>("out_anchors", R"code(Absolute coordinates of LU corner
+If neither `shapes` nor `shapes_rel` are provided, the shape is calculated so that
+the region goes from the region anchor in the input image until the end
+of the input image.)code",
+                         nullptr, true, true)
+    .AddOptionalArg<float>("shapes_rel", R"code(Relative shape of the paste regions.
+
+Works like `shape` argument, but the values should be floats in `[0, 1]` range,
+describing the paste region shape relative to the input shape.)code",
+                           nullptr, true, true)
+    .AddOptionalArg<int>("out_anchors", R"code(Absolute coordinates of LU corner
 of the destination region.
 
 The anchors are represented as 2D tensors where the first dimension corresponds to the
-elements of ``in_ids`` and the second one is equal to the number of dimensions of the
+number of pasted regions and the second one is equal to the number of dimensions of the
 data, excluding channels.
 
-If not provided, all anchors are zero.)code", nullptr, true)
-.AddArg("output_size",
-R"code(Shape of the output.)code", DALI_INT_VEC, true)
-.AddOptionalTypeArg("dtype",
-R"code(Output data type. If not set, the input type is used.)code")
-.NumOutput(1);
+In neither `out_anchors` nor `out_anchors_rel` are provided, all anchors are zero.)code",
+                         nullptr, true, true)
+    .AddOptionalArg<float>("out_anchors_rel", R"code(Absolute coordinates of LU corner
+of the destination region.
+
+Works like `out_anchors` argument, but the values should be floats in `[0, 1]` range,
+describing the LU corner of the pasted region relative to the output shape.)code",
+                           nullptr, true, true)
+    .AddOptionalArg<std::vector<int>>("output_size",
+                                      R"code(A tuple (HW) describing the output shape.
+
+Can be omitted if the operator is run with inputs of uniform shape. In that case,
+the same shape is used as the output shape.)code",
```

Comment:
We can mention `canvas` here, and discuss the output in its terms.

Reply:
Sure, I use canvas when referring to output shape now.